app/src/main/res/layout/layout_apps.xml

```diff
@@ -5,7 +5,9 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:background="@color/new_background"
     android:orientation="vertical"
-    android:duplicateParentState="true">
+    android:duplicateParentState="true"
+    android:paddingTop="140dp"
```

Comment:
Same here requires previous suggestion.

Reply:
Well, the issue is on higher resolution the UI overlaps the actionbar

![Uploading Screenshot_20250417_210956.png…]()